src/js/components/ThumbsRating/__tests__/ThumbsRating.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
Can we make this a typescript file? Trying to make sure any new test files that are added are typescript

Reply:
done